### PR TITLE
add:feature/1 model

### PR DESCRIPTION
**Pull request by Artem-Metlushko**

### CODE REVIEW

File: src/main/java/com/metlushko/strawberry/model/Order.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Order {
+
+    private UUID orderId;
```
Comment (emkurilenko) [inline src/main/java/com/metlushko/strawberry/model/Order.java]: Use Long id

File: src/main/java/com/metlushko/strawberry/model/Product.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Product {
+
+    private UUID id;
```
Comment (emkurilenko) [inline src/main/java/com/metlushko/strawberry/model/Product.java]: Use Long id

File: src/main/java/com/metlushko/strawberry/model/Seller.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Seller {
+
+    private UUID sellerId;
```
Comment (emkurilenko) [inline src/main/java/com/metlushko/strawberry/model/Seller.java]: Use Long id

File: src/main/java/com/metlushko/strawberry/model/User.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class User {
+
+    private UUID userId;
```
Comment (emkurilenko) [inline src/main/java/com/metlushko/strawberry/model/User.java]: Use Long id

File: src/main/java/com/metlushko/strawberry/model/Order.java [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    private List<Product> products;
+    private LocalDate created;
```
Comment (emkurilenko) [inline src/main/java/com/metlushko/strawberry/model/Order.java]: LocalDateTime

File: src/main/java/com/metlushko/strawberry/model/Order.java (hunk 1 above)
Comment (Artem-Metlushko) [reply src/main/java/com/metlushko/strawberry/model/Order.java]: Done

File: src/main/java/com/metlushko/strawberry/model/Product.java (hunk 2 above)
Comment (Artem-Metlushko) [reply src/main/java/com/metlushko/strawberry/model/Product.java]: Done

File: src/main/java/com/metlushko/strawberry/model/Seller.java (hunk 3 above)
Comment (Artem-Metlushko) [reply src/main/java/com/metlushko/strawberry/model/Seller.java]: Done

File: src/main/java/com/metlushko/strawberry/model/User.java (hunk 4 above)
Comment (Artem-Metlushko) [reply src/main/java/com/metlushko/strawberry/model/User.java]: Done

File: src/main/java/com/metlushko/strawberry/model/Order.java (hunk 5 above)
Comment (Artem-Metlushko) [reply src/main/java/com/metlushko/strawberry/model/Order.java]: Done

**Pull request merged**